README.md

```diff
@@ -1,17 +1,148 @@
+
+Define high level categories within the configuration, including:
+
+- **components:** create charts, tables, input/output interfaces, and more
+- **controls**: create filters, parameter inputs, and custom action controllers
+- **pages, layouts and navigation**: create multiple pages, with customisable layouts and flexible navigation across them
+- **actions and interactions**: create interactions between charts, and use pre-defined or customised actions (such as exporting)
+
+Configuration can be written in multiple formats including **Pydantic models**, **JSON**, **YAML** or **Python dictionaries** for added flexibility of implementation
+
+Optional high-code extensions allow almost infinite customisation in a modular way, combining the best of low-code and high-code - for flexible and scalable, Python enabled data visualization applications
+
+<br/>
+
+### Key benefits
+
+<br/>
+
+<p align="center">
+<img src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/value_prop_icons.png" width="900"/>
+</p>
+
+<br/>
+
+### Examples
+
+<p align="center">
+<img src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/dashboard_examples.png" width="1300"/>
+</p>
+
+### Live demo
+
+<p align="left">
+<a href="http://vizro.mckinsey.com/">
+<img border="0" src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/live_interactive_demo.png" width="525" height="296">  </a>
+</p>
+
+View a live <a href="http://vizro.mckinsey.com/" target="_blank">interactive demo </a>
+
+---
+
+## Installation
+
+```console
+pip install vizro
+```
+
+See the [Install guide](https://vizro.readthedocs.io/en/latest/pages/user_guides/install/) for more information
+
+Please note this repository is a monorepo and the core `vizro` package can be found in [/vizro-core](https://github.com/mckinsey/vizro/tree/main/vizro-core)
+
+## Getting started
+
+See the [Tutorials](https://vizro.readthedocs.io/en/latest/pages/tutorials/first_dashboard/) for creating your first dashboard
 
 ## Documentation
 
-See the [`docs`](https://vizro.readthedocs.io/en/latest/) for more details
+See the [Documentation](https://vizro.readthedocs.io/en/latest/) for more details
+
+---
+
+## Community and Development
+
+We encourage you to ask and answer technical questions via the [GitHub Issues](https://github.com/mckinsey/vizro/issues)
+
+## Contributing
+
+To learn more about making a contribution,
+please see the [Contributing Guide](https://vizro.readthedocs.io/en/latest/pages/development/contributing/) for more information
 
 ## Contributors
 
-See the [authors](https://github.com/mckinsey/vizro/blob/main/vizro-core/docs/pages/development/authors.md) file for more details
+View current and former [contributors](https://vizro.readthedocs.io/en/latest/pages/development/authors/)
```

Comment:
Shall we just combine the sections Contributing and Contributors to one Contributing?

Reply:
Yes I like